server/connect.go

```diff
@@ -32,20 +28,17 @@ func (p *Plugin) MaybeSendInviteMessage(userID string) (bool, error) {
 		return false, nil
 	}
 
-	var nWhitelisted int
-	var pendingSince time.Time
-	now := time.Now()
-
 	user, err := p.apiClient.User.Get(userID)
 	if err != nil {
-		p.API.LogWarn("Error getting user", "user_id", userID, "error", err.Error())
-		return false, err
+		return false, errors.Wrapf(err, "Error getting user")
```

Comment:
nit: lower case errors
```suggestion
		return false, errors.Wrapf(err, "error getting user")
```